frontend/src/Games/DTI48/DTI48.tsx

```diff
@@ -1,22 +1,38 @@
 import { useState, useEffect } from 'react';
 import { Loader } from 'semantic-ui-react';
 import { MembersAPI } from '../../API/MembersAPI';
-import DTI48MainGame from './DTI48MainGame';
+import DTI48MainGame, { MembersFromAllSemesters } from './DTI48MainGame';
 import styles from './DTI48.module.css';
 
+async function getMembers(semester?: string): Promise<readonly IdolMember[]> {
+  if (semester == null) return MembersAPI.getAllMembers(true);
+  return fetch(`/members-archive/members-${semester}.json`)
+    .then((response) => response.json())
+    .then((json) => json.members);
+}
+
+async function getMembersFromAllSemesters(): Promise<MembersFromAllSemesters> {
+  const [currentSemesterMembers, sp21Members] = await Promise.all([
+    getMembers(),
+    getMembers('sp21')
+  ]);
+  return {
+    'Current Semester': currentSemesterMembers,
+    'Spring 2021': sp21Members
```

Comment:
Is there a way to not hardcode `sp21Members`, `'sp21'`, and `'Spring 2021'`

Reply:
There is probably a way to make the backend return an API like this, but I plan to address it in another PR where I add the login support for achieved members.